### ISSUE TITLE
Register foreign travel 

### ISSUE THREAD

**Issue by woottone** [cds-snc/booking-use-cases#16]
**Brief description of use case:**
If you're a Canadian citizen travelling abroad, you can register with the government in case of an emergency so they know your location. 

You can register online and in person at a Canadian government office abroad. It seems that unless you have a genuine emergency, you have to book an appointment at consulates. This service would likely fall under the "consulate services" option in [this issue](https://github.com/cds-snc/booking-use-cases/issues/13).

**Service URL:** https://travel.gc.ca/travelling/registration
**End user:** Canadian citizen
**Number of potential end users:** 670,964 annual visits to start pages
**Currently offering appointments:** no
**Where does/could the appointment fit in the service journey?:** They could potentially make an appointment to go and register in person at the office. 
**Does the service require payment?:** no 
**Contact:** NEI
**Service standards that could apply or be affected:** https://travel.gc.ca/assistance/emergency-info/consular/standards

**Comment by smcmurtry** [comment]
@woottone I just took a look at the online form for this: https://www.voyage2.gc.ca/minroca/std/main-en.htm

I'm not sure this is a great candidate for booking an appointment since there's already an online channel that looks like it's easy to use.

**Comment by woottone** [comment]
@smcmurtry yeah I had just included it cause there's the option to do it in person. Same with the EI ones, it's kind of iffy on if it's something we'd actually touch. 

This one I actually discovered something a few minutes ago; registering in person means you have to go to a consulate and from what I can tell you need to make an appointment to go to the consulates that I looked at (they all run on the same system actually). So this would actually fall as a service under the [Canadian Consular in the UK issue](https://github.com/cds-snc/booking-use-cases/issues/13).
